NAudio/Wave/WaveOutputs/DirectSoundOut.cs

```diff
@@ -550,6 +550,41 @@ private void StopPlayback()
             {
                 if (secondaryBuffer != null)

```

Comment:
Thank you for reviewing.
Based on the advice of @markheath , I cleared the secondary buffer clearing process as a helper method at c74756b0c2aa86066a579c8c86f5d442dc154a69 .